niworkflows/interfaces/masks.py

```diff
@@ -267,7 +267,7 @@ def _generate_report(self):
             nsegs = len(levels)
             if nsegs == 0:
                 levels = np.unique(np.round(
-                    nb.load(seg_files[0]).get_data()).astype(int))
+                    nb.load(seg_files[0]).get_fdata(dtype='float32')))
```

Comment:
I'd visually check the results here. I'm a bit worried about not casting to int.

Reply:
Even with the round?

Reply:
Probably fine. It was coerced before for some reason, though...

Reply:
I've checked the ROIPlots interface that uses this and everything checks out :+1: